pcweb/templates/docpage/blocks/code.py

```diff
@@ -117,7 +117,6 @@ def doccmdoutput(
         border_radius=styles.DOC_BORDER_RADIUS,
         border=f"2px solid {rx.color('mauve', 3)}",
         position="relative",
-        margin="1em",
```

Comment:
Do we want to remove, or should this be `margin_y` ?

Reply:
We'll, remove looks fine to me. We could leave the y margin too